reactions/src/SPT.cpp

```diff
@@ -936,7 +937,7 @@ for(int zi = 0; zi<noz; zi++){
 }
 }
 
-// BILL MOD
+//16/03/2021:  Work in progress - needs to be updated to be used in cosmosis
```

Comment:
Is this currently broken? What happens if a user tries to call this via cosmosis -- perhaps it should throw an error or warning if it doesn't already?

Reply:
So Cosmosis shouldn't break but you can only use it for MG only. It won't break since I've allowed the necessary cosmosis wrapper functions to assume a LCDM input P(k) (whereas for massive neutrinos you need the true transfer function to be fed). 

Implementation massive neutrinos to cosmosis will be more involved since we need to feed in the transfer functions for total matter, cb and massive neutrinos for all redshifts which will require us editing the camb module in cosmosis.

Reply:
Ok, thanks.